Description: NTLM and CRAM-MD5 also require creds
 Noticed while writing the DEP8 test for SASL authentication.
Author: Andreas Hasenack <andreas@canonical.com>
Forwarded: https://www.spinics.net/lists/autofs/msg02585.html
Last-Update: 2023-05-24
--- a/modules/lookup_ldap.c
+++ b/modules/lookup_ldap.c
@@ -1208,6 +1208,8 @@
 	if (!strncmp(authtype, "PLAIN", strlen("PLAIN")) ||
 	    !strncmp(authtype, "DIGEST-MD5", strlen("DIGEST-MD5")) ||
 	    !strncmp(authtype, "SCRAM-SHA-", strlen("SCRAM-SHA-")) ||
+	    !strncmp(authtype, "NTLM", strlen("NTLM")) ||
+	    !strncmp(authtype, "CRAM-MD5", strlen("CRAM-MD5")) ||
 	    !strncmp(authtype, "LOGIN", strlen("LOGIN")))
 		return 1;
 #endif
